src/Strategy/Gradle.hs

```diff
@@ -76,17 +79,22 @@ analyze dir = withSystemTempDir "fossa-gradle" $ \tmpDir -> do
       packagePathsWithJson :: [(Text,Text)]
       packagePathsWithJson = map (\line -> let (x,y) = T.breakOn "_" line in (x, T.drop 1 y {- drop the underscore; break doesn't remove it -})) jsonDepsLines
 
-      packagePathsWithDecoded :: [(Text, [JsonDep])]
-      packagePathsWithDecoded = [(name, deps) | (name, outJson) <- packagePathsWithJson
-                                              , Just configs <- [decodeStrict (encodeUtf8 outJson) :: Maybe (Map Text [JsonDep])]
-                                              , Just deps <- [M.lookup "default" configs]] -- FUTURE: use more than default?
-
-      packagesToOutput :: Map Text [JsonDep]
+      packagePathsWithDecoded :: [((Text, Text), [JsonDep])]
```

Comment:
We should probably `newtype` these `Text`s, even though I didn't do it here originally. We can derive `FromJSON` for the configuration name newtype